contracts/AuraLiquidityMigrator.sol

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.11;
+import { IERC20 } from "@openzeppelin/contracts-0.8/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts-0.8/token/ERC20/utils/SafeERC20.sol";
+
+import { IVault, IPriceOracle, IAsset, IBalancerPool, RewardPool, IUniswapV2Pair } from "./Interfaces.sol";
+
+interface IWeightedPoolFactory {
```

Comment:
This file already exists in `mocks/balancer`

Reply:
I followed the pattern of interfaces at `contracts/mocks/balancer/` and use those interfaces instead